tests/smoke/alp_cholesky.cpp

```diff
@@ -171,15 +234,14 @@ void alp_program( const inpdata &unit, alp::RC &rc ) {
 	}
 
 	alp::Matrix< ScalarType, structures::UpperTriangular, Dense > L( N, N );
-	alp::Matrix< ScalarType, structures::Symmetric, Dense > H( N, N );
+	alp::Matrix< ScalarType, HermitianOrSymmetric, Dense > H( N, N );
```

Comment:
UT and Symmetric structures are also square, which means that the Matrix constructor takes only one dimension parameter.